### ISSUE TITLE
Viewing users - mng-list-all & mng-search differences/bug

### ISSUE THREAD

**Issue by cdshow** [lirantal/daloradius#335]
This MAY be my own fault, so I'll give some back story.

I imported data from an older daloradius mysql deployment to a "new one"(newer mysql & web server/php & on a much newer Ubuntu OS).  I performed a cleanup from the old server to the new one at the same time.  I cleaned up the accounting logs to only include data 2 years and newer.  Then I did a SELECT DISTINCT from the accounting table of "users" aka MAC addresses.  Then I did an export of the userinfo table to a csv.  I then used an online tool to covert the CSV back to a SQL insert statement.  I also added entries for each user to radcheck / radusergroups for every mac address.  None of that was too hard or onerous.  Finally the accounting table wasn't working so I found where someone else had a similar issue and it was recommended they dropped the table and re-add it so it had the correct fields.  One item of note is the create table had me create the table as InnoDB.  This is on a new deployment using the guide I had found here as a "basic" outline for what I wanted to do(I wanted to be able to also direct certain MAC addresses to a specific vlan): https://www.missingremote.com/guide/2020/05/setup-freeradius-mysql-daloradius-for-dynamic-vlan-assignment-on-unifi

So I have a total of 3 issues.  First issue is with mng-list-all screen:  If I try to go to page 2+ the page loads as blank.  

Second issue:
I retained the "id" field from the old DB to the new DB.  On the mng-list-all.php first page which does load, all of the id's are missing / do not show up.  

Third issue:
On the mng-search.php the ids DO show up both on the first page and all other pages i.e. page 2, 3 if selected, are "unfiltered."  It is REALLY strange.  Basically, if I search a common name ex. Robert,  John or James, the first page of "results" loads correctly.  If I then select page 2, 3, 4, my results are no longer limited to the "search query" it is now showing what should show up on mng-list-all. BUT the ID shows up fine here.  SO ODD!

I think these entires from the apache error log are related: 
[Thu Jan 12 12:28:52.889560 2023] [php:error] [pid 20767] [client 172.16.xxx.xxx:55599] PHP Fatal error:  Uncaught Error: Undefined constant "ORDER_TYPE_REGEX" in /var/www/html/mng-list-all.php:87\nStack trace:\n#0 {main}\n  thrown in /var/www/html/mng-list-all.php on line 87, referer: http://172.16.xxx.xxx/mng-list-all.php
[Thu Jan 12 12:30:24.586289 2023] [php:error] [pid 20767] [client 172.16.xxx.xxx:55632] PHP Fatal error:  Uncaught Error: Undefined constant "ORDER_TYPE_REGEX" in /var/www/html/mng-list-all.php:87\nStack trace:\n#0 {main}\n  thrown in /var/www/html/mng-list-all.php on line 87, referer: http://172.16.xxx.xxx/mng-list-all.php

To recap simply, the bug(s) are 3 by my count and they may be related and/or self-inflicted.  
1. mng-list won't show additional pages(all pages after page 1 load as blank).
2. mng-list (page 1) does not show the "id" field
3. mng-search does not retain the "filter value" on pages 2+, it becomes unfiltered, but the search DOES show the ID field.  

If I need to, I can zero out all of the tables and start over but...  I thought maybe I screwed something up so I put a fresh install of daloradius in a different subfolder and just copied relevant DB credentials to that config file.  Same issue so... Not sure what the issue is must be DB related or.....?  Any advice or recommendations to fix this?  Is it just my issue, if so, how can I fix it?

**Comment by filippolauria** [comment]
Hi @cdshow , check if #339 fixed some of these issues.

**Comment by cdshow** [comment]
That fixed the first and second issue but the third issue remains.  On a search that spans multiple pages, when you click to go to one of the additional pages, the results are no longer filtered by the search value.  Ex:
This displays fine, http://radius3.sunnyside.cc/mng-search.php?username=james 
This however does NOT display correctly(i.e. clicked to load page 2).  The output is similar to the mng-list-all results i.e. unfiltered, http://radius3.sunnyside.cc/mng-search.php?page=2&orderBy=id&orderType=asc&usernameOnline=james

Thanks again for the first solution, Fix #339  that did resolve 2 of my issues.

**Comment by cdshow** [comment]
PERFECT.  Thanks @filippolauria you have fixed the major issues.  We really appreciate that.  Will both of these fixes be implemented in the next release without "patching"?  Just curious how that will work we want to stay more updated than we did with our old deployment.  We have now moved this deployment into production.  

Two new questions:
When we try to load Last Connection Attempts it starts by showing the last month (so far) is there any way to change the default for Last connection attempts to only the last day of connection attempts?  
Top Users won't load data and we have it set to valid dates.  Any thoughts.  

None of these issues are critical just curious and we can open a new issue if you need us to since the initial issues are resolved.

**Comment by filippolauria** [comment]
> When we try to load Last Connection Attempts it starts by showing the last month (so far) is there any way to change the default for Last connection attempts to only the last day of connection attempts?

At the moment this is the default behavior.

> Top Users won't load data and we have it set to valid dates. Any thoughts.

This looks odd. Share the lines produced in the error.log file of your web server (if any) when visiting the top users page.
